01-getting-started.Rmd

```diff
@@ -21,7 +24,7 @@ options(scipen = 99, digits = 3)
 set.seed(76)
 ```
 
-```{r message=FALSE, warning=FALSE, echo=FALSE}
+```{r message=FALSE, echo=FALSE, purl=FALSE}
```

Comment:
Not sure I agree on setting `purl` to FALSE here. I was thinking that we'd want to share all of the code needed for readers to replicate the results of the R computations in the book in the purled R files.

Reply:
Good point you raise, whoever we define as our target audience has huge ramifications for what gets purled.

The target audience I'm envisioning are readers of the book, but not potential bookdown developers. So we would not purl any code that's not shown to the reader, since they can look that up source code on GitHub.

Let me know what you think.

Reply:
I think you are right. Someone that wants to dig in further can look at the Rmd files to see what's not purled and those users are probably more advanced anyway so we shouldn't be catering to them. How you have done it is more in line with what we say at the end of each chapter too: "An R script file of all R code used in this chapter is available here." Feel free to merge.